build_and_deploy/clients/artifacts_client.ts

```diff
@@ -244,21 +254,18 @@ export class ArtifactClient {
         return new Promise<string>(async (resolve, reject) => {
 
             let url: string = this.buildArtifactUrl(baseUrl, resourceType, payloadObj.name);
-            SystemLogger.info("Url to deploy artifact: " + url);
-
             let payload: string = payloadObj.content;
 
             this.client.put(url, payload, this.getHeaders(token)).then((res) => {
 
                 let resStatus = res.message.statusCode;
-                SystemLogger.info(`ArtifactDeploymentTask status: ${resStatus}; status message: ${res.message.statusMessage}`);
+                SystemLogger.info(`For Artifact: ${payloadObj.name}: ArtifactDeploymentTask status: ${resStatus}; status message: ${res.message.statusMessage}`);
 
                 if (resStatus != 200 && resStatus != 201 && resStatus != 202) {
                     // Remove this after testing
                     res.readBody().then((body) => {
                         if (!!body) {
                             let responseJson = JSON.parse(body);
-                            SystemLogger.info("Deploy artifact failed: " + JSON.stringify(responseJson));
```

Comment:
why we removed this line, at-least print status code?

Reply:
Not sure why was this removed. Yes, it makes sense to keep it there to print the failure